lib/gollum/app.rb

```diff
@@ -464,11 +464,8 @@ class App < Sinatra::Base
         @versions = params[:versions] || []
         if @versions.size == 1
           wikip  = wiki_page(params[:splat].first)
-          adapter_commit = wikip.wiki.repo.commit(@versions.first)
-          impl_commit = adapter_commit.commit
-          impl_parent = impl_commit.parents.first
-          adapter_parent = Gollum::Git::Commit.new(impl_parent)
-          @versions.push(adapter_parent.id)
+          commit = wikip.wiki.repo.commit(@versions.first)
+          @versions.push(commit.parent.id)
```

Comment:
Can `#parent` return `nil`? If so, there might be cases where this statement would try to get the id for `NilClass`.

Reply:
Good point. We need some `nil` checking :)